app/controllers/api/v1/stats/publishers_controller.rb

```diff
@@ -0,0 +1,91 @@
+class Api::V1::Stats::PublishersController < Api::V1::StatsController
+  def signups_per_day
+    sql =
+    """
+      select created_at::date, count(*)
```

Comment:
Is there a way to refactor this so we're not making raw SQL calls?

Reply:
IMO, stats queries are meant to be readable by all parties. Raw SQL calls are the more preferable way to read for non-Rails devs.